src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -230,5 +237,25 @@ bool MulticopterLandDetector::_get_landed_state()
 	return true;
 }
 
+float MulticopterLandDetector::get_takeoff_throttle()
+{
+	/* Position mode */
+	if (_control_mode.flag_control_manual_enabled && _control_mode.flag_control_position_enabled) {
+		/* Should be above 0.5 because below that we do not gain altitude and won't take off.
+		 * Also it should be quite high such that we don't accidentally take off when using
+		 * a spring loaded throttle and have a useful vertical speed to start with. */
+		return 0.9f;
+	}
+
+	/* Manual/attitude mode */
+	if (_control_mode.flag_control_manual_enabled && _control_mode.flag_control_attitude_enabled) {
+		/* Should be quite low and certainly below hover throttle because pilot controls throttle manually. */
+		return 0.15f;
```

Comment:
Why exactly 0.25?
I took 0.15 because of this check that existed before:
https://github.com/PX4/Firmware/pull/6205/files#diff-bcb38f37ec5bebecc13965e3fec8c312R183